spec/rspec/core_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe RSpec do
   fake_libs = File.expand_path('../../support/fake_libs', __FILE__)
   allowed_loaded_features = [
+    /open3.rb/,       # Used by Bisect::Runner, which is not normally loaded
```

Comment:
> Yeah but we have no guarantee of that after this change...

Doesn't particularly bother me.  Ruby is a poor language to use if you care about isolating what's loaded that much.

Still, I thought of a way that provides a stronger guarantee that I think will address your concerns: f5483c45401a397a5b955b737e3b8a18989b5770.  With that in place, if anyone adds a `require 'open3'` to another part of rspec-core, we'll get a spec failure to notify us of the fact.